projects/igniteui-angular/src/lib/grids/grid/grid.component.html

```diff
@@ -53,6 +53,11 @@
                     }">unfold_more</igx-icon>
                 </div>
             </ng-container>
+            <ng-container *ngIf="rowDraggable">
+                <div class="igx-grid__tr--drag-indicator" #headerDragContainer>
+                    <igx-icon fontSet="material" style="visibility: hidden;">drag_indicator</igx-icon>
```

Comment:
I think that here you should take into account that a custom drag handle could also be passed. For example, if a user passes a really long string or large icon as the drag handle, the header will still be sized according to the initial icon, resulting in a broken layout